docs/components/Page.js

```diff
@@ -4,12 +4,21 @@ import { Box, Flex, Link, Text } from 'gestalt';
 import SearchContent from './SearchContent.js';
 import Toc from './Toc.js';
 
+const DETAIL_PAGE_MAX_WIDTH = 1312;
+
 type Props = {|
   children: Node,
   title: string,
+  showSideNav?: boolean,
+  showEditLink?: boolean,
 |};
 
-export default function Page({ title: page, children }: Props): Node {
+export default function Page({
+  title: page,
+  children,
+  showSideNav = true,
```

Comment:
Boolean props defaulting to `true` are a bit of an anti-pattern to me, as the API is then a bit awkward:
```js
<Component booleanProp={false} />
```
Consider inverting the logic to `hideSideNav` and `hideEditLink`, such that the default usage is just
```js
<Page />
```
and the special cases are simpler, like
```js
<Page hideSideNav hideEditLink />
```

The current boolean props read better in the contained markup, but the component API is more important IMO.